resources/storage_class.py

```diff
@@ -7,13 +7,12 @@
 import yaml
 import tempfile
 
-
-from resources.base_resource import BaseOCSClass
+from resources.ocs import OCS
 
 log = logging.getLogger(__name__)
 
 
-class StorageClass(BaseOCSClass):
+class StorageClass(OCS):
```

Comment:
@ebenahar Are we keeping it?

Reply:
@RazTamir Not sure what to do we these files. Let's wait with the deletion for another PR

Reply:
ok